src/components/CreatePostInput/index.js

```diff
@@ -190,9 +177,7 @@
     <>
       <TextInput
         ref={inputRef}
-        onKeyPress={isTopicInDeletedText}
         onChangeText={handlePostTextChanged}
-        // value={message}
         multiline={true}
         numberOfLines={10}
         style={[
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onKeyPress={isTopicInDeletedText}` prop has been removed from the TextInput component. Ensure that this change doesn't affect the expected behavior of the component. If the prop is no longer needed, you can disregard this comment.
```diff
-        onKeyPress={isTopicInDeletedText}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->